feat: allowing forward to offline dep (#31976)
RocketChat/Rocket.Chat: .changeset/strange-rivers-live.md

---
'@rocket.chat/core-typings': minor
'@rocket.chat/i18n': minor
'@rocket.chat/meteor': minor
---

> Enable the feature to allow agents to forward inquiries to the offline department when the parameter "allowReceiveForwardOffline" is set to true for each specific department. This configuration empowers agents to seamlessly direct incoming requests to the designated offline department, ensuring efficient handling of queries even when departmental resources are not actively online. When an agent becomes available online, any pending inquiries will be automatically routed to them.

Reviewer: ```suggestion
Added support for allowing agents to forward inquiries to departments that may not have any online agents given that `allowReceiveForwardOffline` is set to `true` in the department configuration. 
This configuration empowers agents to seamlessly direct incoming requests to the designated department, ensuring efficient handling of queries even when departmental resources are not actively online. When an agent becomes available, any pending inquiries will be automatically routed to them if the routing algorithm supports it.
```

---
'@rocket.chat/core-typings': minor
'@rocket.chat/i18n': minor
'@rocket.chat/meteor': minor
---

> Added support for allowing agents to forward inquiries to departments that may not have any online agents given that `allowReceiveForwardOffline` is set to `true` in the department configuration. 
> This configuration empowers agents to seamlessly direct incoming requests to the designated department, ensuring efficient handling of queries even when departmental resources are not actively online. When an agent becomes available, any pending inquiries will be automatically routed to them if the routing algorithm supports it.

Reviewer 2: ```suggestion
Added the ability for agents to seamlessly direct incoming requests to the designated department, ensuring efficient handling of queries even when departmental resources are not actively online. When an agent becomes available, any pending inquiries will be automatically routed to them if the routing algorithm supports it.
```

User oriented 🙈

Reviewer 2: Also how agents can use it (a toggle? a button in ui? etc) in short would be nice as well.
